src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,178 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.RegistrationValidationExeption;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final long FIRST_TEST_ID = 12345;
+    private static final long SECOND_TEST_ID = 54321;
+    private static final int AGE_18_YEARS = 18;
+    private static final int AGE_LES_18_YEARS = 15;
+    private static final int AGE_MORE_18_YEARS = 21;
+    private static final int AGE_MORE_100_YEARS = 105;
+    private static final String FIRST_LOGIN_OK = "TuttiFrutti";
+    private static final String SECOND_LOGIN_OK = "FruttiTutti";
+    private static final String LOGIN_CONTAINS_SPACE = "Gringo gringo";
+    private static final String PASSWORD_8_SYMBOL_OK = "buffer789";
+    private static final String PASSWORD_8_SYMBOL_SECOND_OK = "grinGo666";
+    private static final String PASSWORD_LESS_THEN_8_OK = "notwork";
+    private static StorageDao storageDao;
+    private static User firstUser;
+    private static User secondUser;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        firstUser = new User();
+        firstUser.setId(FIRST_TEST_ID);
+        firstUser.setLogin(FIRST_LOGIN_OK);
+        firstUser.setAge(AGE_MORE_18_YEARS);
+        firstUser.setPassword(PASSWORD_8_SYMBOL_OK);
+
+        secondUser = new User();
+        secondUser.setId(SECOND_TEST_ID);
+        secondUser.setLogin(SECOND_LOGIN_OK);
+        secondUser.setAge(AGE_18_YEARS);
+        secondUser.setPassword(PASSWORD_8_SYMBOL_SECOND_OK);
+    }
+
+    @AfterEach
+    void clear() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullUser_NotOk() {

```

Comment:
```suggestion
    void register_nullUser_notOk() {

```
the third part should start with a lowercase letter, fix it in all places

Reply:
agree with @mishok321 , let's fix your namings